### PR TITLE
Fail open on 5xx from mixer or missing mixer clusters

### PR DESCRIPTION
**Pull request by rshriram**

### CODE REVIEW

**Comment by qiwzhang** [comment]
Please use clang-format 3.8.0 to format the c++ code.   It is too much change.

Checking mixer failure is done here

https://github.com/istio/mixerclient/blob/master/include/options.h#L46

I think this change should be in mixerclient too.

**Comment by rshriram** [comment]
I used clang-format-3.8 -style=Google and this is what it gave me.

File: src/envoy/mixer/utils.cc [hunk 1]
```diff
@@ -13,6 +13,7 @@
  * limitations under the License.
  */
 
+#include "common/http/headers.h"
```
Comment (lizan) [inline src/envoy/mixer/utils.cc]: This should go after `#include "src/envoy/mixer/utils.h"` (also add an empty line), and breaks style check.

**Comment by qiwzhang** [comment]
Here is the mixer client change:  https://github.com/istio/mixerclient/pull/96

**Comment by rshriram** [comment]
@qiwzhang can I close this, assuming you have fixed the client?

**Pull request closed without merging**